src/mon/AuthMonitor.cc

```diff
@@ -1929,6 +1876,145 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
   return false;
 }
 
+/* Generate the caps that should be present in the entity's auth keyring
+ * after running the "fs authorize" command. This is done by merging the
+ * caps already present in the client's auth keyring with the new caps
+ * provided by the user at "fs authorize" command.
+ */
+bool AuthMonitor::_gen_wanted_caps(EntityAuth& e_auth,
+  vector<string>& newcaps, ostream* out)
+{
+  bool all_caps_same = true; // whether all caps passed are same as the
+			     // one present.
+
+  if (e_auth.caps.empty()) {
+    all_caps_same = false;
+    return all_caps_same;
+  }
+
+  for (auto it = newcaps.begin(); it != newcaps.end(); it += 2) {
+    string& cap_entity = *it;
+    string& n_cap_str = *(it + 1); // new cap to be added to the current cap
+
+    if (e_auth.caps.count(cap_entity) == 0) {
+      all_caps_same = false;
+      continue;
+    }
+
+    string c_cap_str; // current cap held by entity's auth keyring
+    auto i = e_auth.caps[cap_entity].cbegin();
+    decode(c_cap_str, i);
+    if (c_cap_str == n_cap_str) {
+      continue;
+    }
+
+    if (cap_entity == "mon") {
+      MonCap c_cap, n_cap;
+      if (not c_cap.parse(c_cap_str, out))
```

Comment:
nit: enclose within `{}`

Reply:
The code under `if` was a single line, so it wasn't needed.

In latest push, it has multiple line, so it's enclosed now.

Reply:
> The code under `if` was a single line, so it wasn't needed.

Convention is to enclose even a single line within `{}`